projects/examples/src/components/action-menu/action-menu.example.component.ts

```diff
@@ -0,0 +1,198 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { Component } from '@angular/core';
+import { ActionDisplayConfig, ActionItem, ActionStyling, ActionType, TextIcon } from '@vcd/ui-components';
+
+interface Record {
+    value: string;
+    paused: boolean;
+}
+
+interface HandlerData {
+    foo: string;
+    bar: string;
+}
+
+@Component({
+    selector: 'vcd-datagrid-link-example',
+    template: `
+        <div>
+            <button (click)="changeStaticActionStyling()" class="btn btn-primary">
+                Display static actions
+                {{ actionDisplayConfig.staticActionStyling === 'INLINE' ? 'dropdown' : 'inline' }}
+            </button>
+            <br />
+            <vcd-action-menu
+                [actions]="staticActions"
+                [actionDisplayConfig]="actionDisplayConfig"
+                [selectedEntities]="selectedEntities"
+                [dropdownTriggerBtnText]="'vcd.cc.action.menu.actions'"
+            >
+            </vcd-action-menu>
+        </div>
+
+        <br />
+
+        <div>
+            <button (click)="changeContextualActionStyling()" class="btn btn-primary">
+                Display contextual actions
+                {{ actionDisplayConfig.contextual.styling === 'INLINE' ? 'dropdown' : 'inline' }}
+            </button>
+            <button (click)="toggleDropdownDisable()" class="btn btn-primary">
+                {{ isDropdownDisabled ? 'Enable dropdown' : 'Disable dropdown' }}
+            </button>
+            <br />
+            <vcd-action-menu
+                [actions]="contextualActions"
+                [actionDisplayConfig]="actionDisplayConfig"
+                [selectedEntities]="selectedEntities"
+                [dropdownTriggerBtnText]="'vcd.cc.action.menu.actions'"
+                [disabled]="isDropdownDisabled"
+            >
+            </vcd-action-menu>
+        </div>
+    `,
+})
+export class ActionMenuExampleComponent<R extends Record, T extends HandlerData> {
+    actions: ActionItem<R, T>[] = [
+        {
+            textKey: 'Static Featured 1',
+            handler: () => console.log('Static Featured 1'),
+            availability: () => true,
+            actionType: ActionType.STATIC_FEATURED,
+            isNotTranslatable: true,
```

Comment:
Let's avoid the double negative. You can either:

* Name it `isTranslatable` and default it to true if its value is undefined wherever it's used. It's only used in a single place but would get ugly if we have to read it in multiple places
* Make it a class instead of an interface and default it from the constructor
* Rename it to `isRawString`